www/src/components/shell/TerminalInformation.js

```diff
@@ -82,16 +86,20 @@ function TerminalInformation({ shell }) {
               name="Git"
               value={shell.gitUrl}
             />
+            <Divider
+              backgroundColor="border"
+              marginTop="small"
+            />
             <ShellHint
               name="Sync with your local machine"
-              code="plural shell sync"
-              hint="this will clone your repo locally, and sync all decryption keys"
+              code="plural login && plural shell sync"
```

Comment:
if you aren't already logged in, `plural shell sync` actually will do that for you now

Reply:
Done, PTAL.